src/ls.js

```diff
@@ -128,3 +149,16 @@ function _ls(options, paths) {
   return list;
 }
 module.exports = _ls;
+
+
+function ls_stat(path) {
+  var stats = fs.statSync(path);
+  // Note: this object will contain more information than .toString() returns
+  var long_stats = Object.create(stats);
```

Comment:
One slight drawback to using `Object.create()` is that the stats are on the object's prototype, which makes them invisible in a console.log of the object. 

It might be better to append the `stats` properties directly to `long_stats` to make things slightly clearer for debugging.